### ISSUE TITLE
This site can’t be reached (on macOS)

### ISSUE THREAD

**Issue by kozo2** [plotly/dashR#182]
I tried Rscript app.R with https://gist.github.com/kozo2/5f8a7737c8a41988509f9ef2f491c295
on
```
> sessionInfo()
R version 3.6.2 (2019-12-12)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.3

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] C/UTF-8/C/C/C/C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.6.2
```

But http://127.0.0.1:8050/ returns 

<img width="822" alt="Screen Shot 2020-02-27 at 2 15 07" src="https://user-images.githubusercontent.com/12192/75369567-0b73c100-5907-11ea-8028-57eed764246b.png">

It seems there is no error in the terminal

```
~ Rscript app.R
 要求されたパッケージ ggplot2 をロード中です 

 次のパッケージを付け加えます: ‘plotly’ 

 以下のオブジェクトは ‘package:ggplot2’ からマスクされています: 

     last_plot 

 以下のオブジェクトは ‘package:stats’ からマスクされています: 

     filter 

 以下のオブジェクトは ‘package:graphics’ からマスクされています: 

     layout 

```

Do you have any idea about this?

**Comment by rpkyle** [comment]
Hi @kozo2! I notice that you have `0.0.0.0` set for the host IP in line 33:

```
app$run_server(host="0.0.0.0")
```

Could you try using that address instead of `127.0.0.1` and let me know if it resolves the issue?

**Comment by rpkyle** [comment]
Hi @kozo2! My initial reasoning was not too helpful, but to help debug this (since your code produces a webpage which loads for me), is it possible to tell me what value is returned when you type the following?

```
> app$server$port
```

**Comment by kozo2** [comment]
Sorry I forgot to update the gist.
I used `app$run_server()` in the first trial.
And now I added the
```
print(app$server$port)
```
It prints
```
[1] 8080
```
and also checked opening not only `127.0.0.1:8050 localhost:8050` but also `127.0.0.1:8080 localhost:8080`.

But it still shows `This site can’t be reached`.

BTW
https://gist.github.com/kozo2/5f8a7737c8a41988509f9ef2f491c295
works well in Ubuntu.
I will try the same thing also in Windows.
